aws_lambda_events/Cargo.toml

```diff
@@ -75,3 +75,5 @@ rabbitmq = []
 ses = []
 streams = []
 generated = ["activemq", "appsync", "autoscaling", "chime_bot", "clientvpn", "code_commit", "codebuild", "codedeploy", "codepipeline_cloudwatch", "codepipeline_job", "config", "connect", "ecr_scan", "firehose", "iot", "iot_1_click", "iot_button", "kafka", "lambda_function_urls", "lex", "rabbitmq", "ses", "streams"]
+
+url-query = ["query_map/url-query"]
```

Comment:
I've updated it.

Digging a bit, it looks like that syntax has been supported for quite a while. I found [this reference](https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md#cargo-143-2020-04-23) to a fix related to the handling of that part of `Cargo.toml` in v1.43 (2020-04-23). So it's been supported since sometime prior to that.

There was some [new syntax added to that part of `Cargo.toml` in v1.60](https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md#cargo-160-2022-04-07), though (`dep:` and `?`).